config/manifests/bases/self-node-remediation.clusterserviceversion.yaml

```diff
@@ -117,7 +117,7 @@ spec:
   - email: medik8s@googlegroups.com
     name: Medik8s Team
   maturity: alpha
-  minKubeVersion: 1.20.0
+  minKubeVersion: 1.25.0
```

Comment:
this will break OCP 4.10/4.11, not?

Reply:
I believe so, that's part of the reason it's WIP.
I wanted to have some discussion on what's the best way to go about it.